inst/include/interface/README.md

```diff
@@ -0,0 +1,7 @@
+## Interface
+
+This directory contains the Rcpp interface files for the FIMS C++ code. The interface files are responsible for exposing the C++ functionality to R, allowing users to call C++ functions and access C++ classes directly from R. The interface files typically include function definitions, class declarations, and other necessary components to facilitate communication between R and C++. By using Rcpp package available within R, we can leverage the performance benefits of C++ while still providing a user-friendly interface for R users.
```

Comment:
This directory contains interface files for TMB and Rcpp with Rcpp specific interface files in the rcpp directory while init.hpp and interface.hpp contain code that interfaces with TMB. This structure may change with the TMBad-RTMB changes so we will want to update this text when that happens.

Reply:
Sounds good, we can address that once we get the last few 🐞 🪲 worked out of that branch.